### ISSUE TITLE
[Issue] ISN'T Breaking Ender dragon rendering

### ISSUE THREAD

**Issue by Saereth** [nanite/ItShallNotTick#27]
### Minecraft Version

1.19.2

### Modloader

43.2.4

### Mod version

ItShallNotTick-1.0.11-build.35.jar

### Repeatable?

- [X] Yes

### Link to log files

can provide a fresh one if you need

### Steps to reproduce the issue

This is reproducible with a pretty minimal modset I just ran Top, JEI and ISN'T with default settings. Ender dragon is on the ignore list 

`
	entityIgnoreList = ["minecraft:wither", "minecraft:phantom", "minecraft:ender_dragon", "minecraft:elder_guardian", "minecraft:player"]`

### Description of the issue you are having

Basically what happens is once the ender dragon flies more than like 64 blocks away it stops rendering. If you remove ISNT this no longer occurs. 

Possibly related to https://github.com/nanite/ItShallNotTick/issues/12 ?

If you need anymore info you can just ping me and I'll get whatever I you need

### Information

- [X] I have provided as much information as possible
